mage_integrations/mage_integrations/destinations/oracledb/__init__.py

```diff
@@ -0,0 +1,216 @@
+        return [
+            'SELECT name FROM v$database',
+        ]
+
+    def test_connection(self) -> None:
+        conn = self.build_connection().build_connection()
+        cursor = conn.cursor()
+        try:
+            cursor.execute("""SELECT name FROM v$database""")
+        except Exception as exc:
+            self.logger.error(f"test_connection exception: {exc}")
+            raise exc
+        return
+
+    def build_create_table_commands(
+        self,
+        schema: Dict,
+        schema_name: str,
+        stream: str,
+        table_name: str,
+        database_name: str = None,
+        unique_constraints: List[str] = None,
+    ) -> List[str]:
+        return [
+            build_create_table_command(
+                column_type_mapping=column_type_mapping(
+                    schema,
+                    convert_column_type,
+                    lambda item_type_converted: 'VARRAY(5)',
+                ),
+                columns=schema['properties'].keys(),
+                full_table_name=f'{table_name}',
+                key_properties=self.key_properties.get(stream),
+                schema=schema,
+                unique_constraints=unique_constraints,
+            ),
+        ]
+
+    def build_alter_table_commands(
+        self,
+        schema: Dict,
+        schema_name: str,
+        stream: str,
+        table_name: str,
+        database_name: str = None,
+        unique_constraints: List[str] = None,
+    ) -> List[str]:
+        results = self.build_connection().load(f"""
+SELECT
+     column_name
+    , data_type
+FROM USER_TAB_COLUMNS
+WHERE TABLE_NAME = '{table_name.upper()}'
+""")
+        current_columns = [r[0].lower() for r in results]
+        schema_columns = schema['properties'].keys()
+        new_columns = [c for c in schema_columns
+                       if clean_column_name(c, handle_leading_underscore=False).lower()
+                       not in current_columns]
+
+        if not new_columns:
+            return []
+        return [
+            build_alter_table_command(
+                column_type_mapping=column_type_mapping(
+                    schema,
+                    convert_column_type,
+                    lambda item_type_converted: 'TEXT',
+                ),
+                columns=new_columns,
+                full_table_name=f'{table_name}',
+            ),
+        ]
+
+    def build_insert_commands(
+        self,
+        records: List[Dict],
+        schema: Dict,
+        schema_name: str,
+        table_name: str,
+        database_name: str = None,
+        unique_conflict_method: str = None,
+        unique_constraints: List[str] = None,
+    ) -> List[str]:
+        columns = list(schema['properties'].keys())
+        insert_columns, insert_values = build_insert_command(
+            column_type_mapping=column_type_mapping(
+                schema,
+                convert_column_type,
+                lambda item_type_converted: 'LONGTEXT',
```

Comment:
again, this method should be the same with the one used in the `build_create_table_commands` and `build_alter_table_commands` methods.

Reply:
Updated it to be consistent with LONG type